### DISCUSSION TITLE
[Solved] How to contribute to zigbee2mqtt guide?

### DISCUSSION THREAD

**Discussion by Entepotenz** [Koenkk/zigbee2mqtt#14342] [General]
Hi everyone,

I would like to add some information to the zigbee2mqtt guide: https://www.zigbee2mqtt.io/guide/supported-hardware.html

In which repository can I find the guide text files?

Best regards :-)

**Comment by Nardol** [comment]
On the bottom of the page, there is a link to edit it on Github.
This will create a fork on your Github account and you'll be able to make a pull request with your modifications.

**Reply by Entepotenz** [reply]
Thanks. I am blind 🙈
